tools/python/run.py

```
@@ -22,76 +23,226 @@
+        print(f"Using default MIMO config: {ns.config}")
+    
+    print("MIMO: Multiple Inputs → Multiple Outputs")
+    print("Dataset: MIMO-benchmark (3 inputs → 3 outputs)")
+    print("Expressions: [x1*x2, sin(x3), x1+x2*x3]")
+    print("="*60)
+    
+    # For MIMO, we need to use the fixed implementation
+    os.chdir(PROJECT)
+    cmd = [python_exe(), "-c", f"""
+import sys
+sys.path.insert(0, {repr(str(DSO_PKG))})
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)